parent/pom.xml

```diff
@@ -190,6 +216,8 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-gpg-plugin</artifactId>
+                        <!-- Older versions have issues with the gpg passphrase -->
+                        <version>1.6</version>
```

Comment:
The same here. But as we know that <1.6 won't work, it makes sense to have an exception for GPG plugion.

Reply:
Right, the comment I made with the nexus-staging-maven-plugin version is also applicable: most projects don't need to configure this plugin at all.